### PR TITLE
Move devDependencies to right place

### PR DESCRIPTION
**Pull request by haskellcamargo**
This PR changes `package.json` and generated `yarn.lock`. I've moved the compile-time packages to `devDependencies`.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,12 +25,20 @@
...
   },
   "devDependencies": {
+    "babel": "^6.5.2",
+    "babel-core": "^6.22.1",
+    "babel-jest": "^18.0.0",
+    "babel-loader": "^6.2.10",
+    "babel-polyfill": "^6.22.0",
     "babel-preset-es2015-native-generators": "^6.6.0",
-    "eslint": "3.13.1",
+    "eslint": "^3.14.1",
```
Comment (derekstavis) [inline package.json]: maybe remove all carets?

Comment (MarcoWorms) [inline package.json]: +1

**Comment by haskellcamargo** [comment]
Please, clone my branch locally and test before merging. I'm having some problems to configure the environment here.

**Comment by derekstavis** [comment]
Whoa, awesome!! :tada:

I've checkouted your branch and it works for me. Thanks for it!

I'm merging as it LGTM :+1:

**Pull request merged**